_sass/tiendatek-theme.scss

```diff
@@ -683,6 +692,113 @@ footer {
   font-family: 'MontserratRegular';
 }
 
+// Mobile view
```

Comment:
IMO, you should include another Sass file (`@import "_tiendatek-theme-mobile.scss"`), instead of using a global comment (clean code!)

Reply:
👍